ragen/env/search/config.py

```
@@ -0,0 +1,51 @@
+from ragen.env.base import BaseEnvConfig
+from dataclasses import dataclass, field
+from typing import Optional, List, Dict
```

Ruff (0.14.10)

`typing.List` is deprecated, use `list` instead

(UP035)